example/deployment.go

```diff
@@ -0,0 +1,65 @@
+package example
+
+import (
+	"fmt"
+
+	"github.com/Azure/draft/pkg/deployments"
+	"github.com/Azure/draft/pkg/templatewriter"
+	"github.com/Azure/draft/pkg/templatewriter/writers"
+	"github.com/Azure/draft/template"
+)
+
+// WriteDeploymentFiles generates Deployment Files using Draft, writing to a Draft TemplateWriter. See the corresponding draft.yaml file for the template inputs.
```

Comment:
maybe include full file path here for those unfamiliar with how draft works/the repo structure?